extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -2052,6 +2052,111 @@ describes.realWin('Events', {amp: 1}, (env) => {
             totalVisibleTime: 15,
           });
         });
+
+        it('should fire event for all elements of the selector', async () => {
```

Comment:
what `events.js` does is take an array of element and register visibility triggers. I'd say we don't need test coverage for `fire event for all elements of the selector` and `fire event per selector`. 
A single `fire event for multiple elements` should work. What do you think?

Reply:
Yea I agree, I added it initially just to be diligent :)